01_fib/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(5)
+
+}
+
+func fib(n int) int {
+	var n1, n2, n3 = 0, 1, 0
```

Comment:
Use `:=` instead of `var` here.

Reply:
thanks for the suggestion, got the difference between := and var, 
in Go, := is for declaration + assignment, whereas = is for assignment only